packages/reference/src/common/MultipleReferenceEditor.tsx

```diff
@@ -90,7 +90,9 @@ function Editor(props: EditorProps) {
     editorPermissions,
     onCreate,
     onLink,
+    itemsLength: items.length,
```

Comment:
Couldn't `itemsLength` be read in the `onLink` and `onCreate` handlers above instead of pass it down to the children?
Something like 
```
const onCreate = useCallback(
    (id: string, index?: number = itemsLength) => onLinkOrCreate(setValue, entityType, items, [id], index),
    [setValue, items, entityType]
  );
```